cnv/cnv_node_network/cnv-troubleshooting-node-network.adoc

```diff
@@ -0,0 +1,15 @@
+[id="cnv-troubleshooting-node-network"]
+= Troubleshooting node network configuration
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-troubleshooting-node-network
+toc::[]
+
+If the node network configuration encounters an issue, the Policy is automatically rolled back and the Enactments report failure.
+This includes issues such as:
+
+* The configuration fails to be applied on the host
+* Lost connection to the default gateway
+* Lost connection to the API server
```

Comment:
I think these need to be parallel in structure (either sentences or fragments) and end in periods.